test/integration/services/price.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { PriceService } from "@services/price"
+import { PriceRange, PriceInterval } from "@domain/price"
+
+describe("Price", () => {
+  describe("listHistory", () => {
+    const rangesIntervalsToTest = [
+      { range: PriceRange.OneDay, interval: PriceInterval.OneHour, ticks: 23 },
+      { range: PriceRange.OneWeek, interval: PriceInterval.FourHours, ticks: 42 },
+      { range: PriceRange.OneMonth, interval: PriceInterval.OneDay, ticks: 30 },
+      { range: PriceRange.OneYear, interval: PriceInterval.OneWeek, ticks: 52 },
+    ]
+    rangesIntervalsToTest.forEach(({ range, interval, ticks }) => {
+      it(`returns ${range} history with ${interval} interval`, async () => {
+        const prices = await PriceService().listHistory({ range, interval })
```

Comment:
is this mocked?

Reply:
only in unit test history is mocked (but that is only for test cache behavior) but here is query directly from db (data is generated in `01-price-history.spec.ts`)